dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -704,12 +877,74 @@ def _get_variance(self, df_series, prev_dependent_properties,
         batch_count = subset_properties["match_count"]
         batch_mean = 0. if not batch_count else \
             float(sum_value) / batch_count
+        subset_properties["mean"] = batch_mean
         self.variance = self._merge_variance(self.match_count, self.variance,
                                              prev_dependent_properties["mean"],
                                              batch_count,
                                              variance,
                                              batch_mean)
 
+    @BaseColumnProfiler._timeit(name = "skewness")
+    def _get_skewness(self, df_series, prev_dependent_properties,
+                      subset_properties):
+        """
+        Computes and updates the skewness of the current dataset given
+        new chunk
+
+        :param df_series: incoming data
+        :type df_series: pandas series
+        :param prev_dependent_properties: pre-update values needed
+            for computation
+        :type prev_dependent_properties: dict
+        :param subset_properties: incoming data statistics
+        :type subset_properties: dict
+        :return None
+        """
+        batch_biased_skewness = skew(df_series)
```

Comment:
typically we want to leave in the fact that it is from scipy, hence  we can instead at the top just `import scipy` and call `scipy.stats.skew` as we don't need the modularity of `skew`

Reply:
If two approvals aren't made by the time this can be updated, we can fix it in this PR, otherwise we will add it to the subsequent PR updating for allowing the user to specify bias correction or not.